docs/design-system.md

```diff
@@ -0,0 +1,73 @@
+# Design System
+
+## Overview
+
+Web Stories exists as two apps right now, the editor and the dashboard. Both have their own set of components that they are built off of. Now that we have a robust design system in figma ([here](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=1906%3A0)), we should stop duplicating foundational and base components by having separate copies in the editor and dashboard. The idea here is to make this transition as painless as possible, no massive structure changes just consolidating components and themes.
+
+## What goes in the design system?
+
+The Design System is **only** for shared basics and components. In [atomic design terms](https://bradfrost.com/blog/post/atomic-web-design) we are sharing atoms, molecules and (some) organisms.  If it is a small pattern that’s repeated in a more complicated structure (like a menu that’s used in a dropDown, a search component and a popover menu) then that structure should exist in the design system. If it’s shared across both the editor and the dashboard, it should exist in the design system - like a button or a dialog. If it’s something complex only used by the editor, say the color picker, which uses a slider and some other smaller components in the design system, the color picker would live in the editor because it is not shared by the dashboard. Things that are not easily generalized should not live in the design system.
+
+## Theme
+
+With this update we get a full new color palette and theme. The structure is changing a little bit to try and keep things more organized. You can dig into the app [assets/src/design-system/theme](https://github.com/google/web-stories-wp/tree/main/assets/src/design-system) to see organization. Here’s brief overview of the organization to contextualize what you’re looking at.
+
+1. Colors, Typography, Borders, Breakpoints are in separate files and all added to the index’s `theme` that can be ingested at the root of each app. This file separation makes it easier to find theme values for developers who prefer to look in source code for the value they need.
+2. The new theme provides a dark and a light variant. The dark is the default. All colors are available regardless of variant, but the `theme.colors.bg` and `theme.colors.fg` are what you should gravitate towards using because the light and dark theme colors mirror each other. This will make dark and light mode functionality easier to handle down the road.
```

Comment:
I think this should be "Dark is the default" or "The dark variant is the default"
```suggestion
2. The new theme provides a dark and a light variant. Dark is the default. All colors are available regardless of variant, but the `theme.colors.bg` and `theme.colors.fg` are what you should gravitate towards using because the light and dark theme colors mirror each other. This will make dark and light mode functionality easier to handle down the road.
```